packages/apps/src/utils/activity-utils.ts

```diff
@@ -0,0 +1,16 @@
+import { ActivityBuilder, ActivityParams, isActivityBuilder, MessageSendActivityBuilder } from '@teams.sdk/api';
+import { Card, isCard } from '@teams.sdk/cards';
+
+export function toActivityParams(activity: ActivityParams | string | ActivityBuilder | Card): ActivityParams {
```

Comment:
why is this one not market as internal but the other two for cards and activity builder are?

Reply:
oversight, will fix :)